JavaScriptTasks/Session 3 - Js_DynamicScope(4-03-2020)/ClosuresAndScopes.js

```diff
@@ -0,0 +1,18 @@
+// PROBLEM STATEMENT :Closures and Scopes
+
+// http://www.codewars.com/kata/closures-and-scopes/train/javascript 
+
+// SOLUTION:
+
+function createFunctions(n) {
+    var callbacks = [];
+  
+    for (var i=0; i<n; i++) {
+    let x = i;
```

Comment:
Remove this unnecessary assignment:  let x = i;

Reply:
removed x and modified code as : 

function createFunctions(n) {
  var callbacks = [];

  for (let i=0; i<n; i++) {
    callbacks.push(function() {
      return i;
    });
  }
  return callbacks;
}